### PR TITLE
Feature/add tutorial

### PR DESCRIPTION
**Pull request by creisle**
Improvements
- Documentation: Added mini tutorial

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+This should output lines similar to the following
+
+```text
+resistance gefitinib [C1855] CIViC
+likely pathogenic lung cancer [DOID:1324] DoCM
+```
+
+## Categorizing Statements
+
+Something we often what to do is to know if a statement is therapeutic, or prognostic, etc. The
```
Comment (mattdoug604) [inline docs/tutorial.md:108]: typo: "Something we often what to do is to know"

File: docs/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+naive approach is the base this on a list of known terms or a regex pattern. In GraphKB we can
```
Comment (mattdoug604) [inline docs/tutorial.md:109]: typo: "is the base" -> "is to base"

File: docs/tutorial.md (hunk 1 above)
Comment (creisle) [reply docs/tutorial.md]: fixed!

File: docs/tutorial.md (hunk 2 above)
Comment (creisle) [reply docs/tutorial.md]: fixed!

**Pull request merged**